packages/eslint-plugin-khan/src/rules/ts-no-error-suppressions.ts

```diff
@@ -0,0 +1,91 @@
+import {ESLintUtils, TSESTree} from "@typescript-eslint/utils";
+
+const createRule = ESLintUtils.RuleCreator(
+    (name) =>
+        `https://github.com/Khan/wonder-stuff/blob/main/packages/eslint-plugin-khan/docs/${name}.md`,
+);
+
+type Options = ["always" | "never"];
+type MessageIds = "errorString";
+
+const message = "{{type}} not allowed on JSXAttributes";
```

Comment:
Interesting. It will have reference to the type in question?

Reply:
The `{{type}}` gets replaced by the `data.type` provide by the `context.report({...})` call at the bottom of the file.  Unfortunately the docs aren't great so I've mainly been learning by looking at some of the rules in https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin/src/rules.

Reply:
Here's some snippets that were helpful in figuring this out:
- https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/rules/explicit-member-accessibility.ts#L59-L65 
- https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/rules/explicit-member-accessibility.ts#L298-L302